app/src/main/java/grinnell/appdev/edu/lyles/fragments/BeersAdaptor.java

```diff
@@ -0,0 +1,111 @@
+package grinnell.appdev.edu.lyles.fragments;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.os.AsyncTask;
+import android.support.annotation.NonNull;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.MalformedURLException;
+import java.util.ArrayList;
+
+import grinnell.appdev.edu.lyles.R;
+
+/**
+ * Created by nannan on 2017/2/12.
+ */
+
+public class BeersAdaptor extends ArrayAdapter<Beers> {
+
+    ArrayList<Beers> ArrayListBeers;
+    int Resource;
+    Context context;
+    LayoutInflater vi;
+
+    public BeersAdaptor(Context context, int resource, ArrayList<Beers> objects) {
+        super(context, resource, objects);
+
+        ArrayListBeers = objects;
+        Resource = resource;
+        this.context=context;
+
+        vi = (LayoutInflater) context.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
```

Comment:
If you change the type of `context` to `Activity` then you can just use the `getLayoutInflater` method.